src/redux/books/books.js

```diff
@@ -0,0 +1,25 @@
+const ADD_BOOK = 'ADD_BOOK';
+const DELETE_BOOK = 'DELETE_BOOK';
```

Comment:
- [ ] Great job so far, as stated on the requirement please use the [ducks pattern](https://github.com/erikras/ducks-modular-redux) when defining action types here and also on the categories page 👍.

![image](https://user-images.githubusercontent.com/59416136/192393097-74783d09-5735-43a4-8517-73b0651ac6ac.png)